lib/faker/default/world_cup.rb

```diff
@@ -3,18 +3,56 @@
 module Faker
   class WorldCup < Base
     class << self
+      ##
+      # Produces a national team name.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::WorldCup.team #=> "Iran"
+      #
+      # @faker.version next
```

Comment:
The Faker::WorldCup class has been around since 1.9.0, so this (and the other methods) should be

```suggestion
      # @faker.version 1.9.0
```